FBSnapshotTestCase/UIImage+Compare.m

```diff
@@ -76,15 +84,53 @@ - (BOOL)compareWithImage:(UIImage *)image
     free(imagePixels);
     return NO;
   }
-    
+
+    /*
+  CGFloat scaleFactor = [[UIScreen mainScreen] scale];
+  CGContextScaleCTM(referenceImageContext, scaleFactor, scaleFactor);
+  CGContextScaleCTM(imageContext, scaleFactor, scaleFactor);
+    */
```

Comment:
Right, we don't need to scale this here since `CGImageGetWidth()` and `CGImageGetHeight()` will return the right value. I wonder why we had this before?